.golangci.yml

```diff
@@ -0,0 +1,27 @@
+run:
+  deadline: 5m
+
+linters:
+  disable-all: true
+  enable:
```

Comment:
Is `gotypex` covered by `typecheck`?

Reply:
I'm not sure...
typecheck - Like the front-end of a Go compiler, parses and type-checks Go code.
gotype -x - Syntactic and semantic analysis in external test packages (similar to the Go compiler).

Reply:
External test packages are `*_test.go` files which are not in the same package as the original file (for example, if you have `fs` package and the test is in `fs_test` package).
Could you check if `typecheck` reports type errors in such files?